examples/bspump-avro-serialization.py

```diff
@@ -1,10 +1,7 @@
 #!/usr/bin/env python3
 import logging
-import json
-import io
-
-import fastavro
-
+from bspump.avro.serializer import AvroSerializer
+from bspump.avro.deserializer import AvroDeserializer
```

Comment:
`import bspump.avro` ... follow the example bellow, we don't use `from ... import ...` outside of the main package for a readability purposes.